qutebrowser/browser/browsertab.py

```diff
@@ -107,6 +108,7 @@ class TabData:
     override_target = attr.ib(None)
     pinned = attr.ib(False)
     fullscreen = attr.ib(False)
+    input_mode = usertypes.KeyMode.normal
```

Comment:
This sets a class variable - it doesn't do anything bad here as it's a constant, but it should still use `attr.ib` like above (see the [attrs documentation](http://www.attrs.org/en/stable/)).